canonical-web-and-design/ubuntu.com
# Update managed kafka

## Done
Update managed kafka

## QA
- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/managed/kafka
- Check the copy matchs the [copy doc](https://docs.google.com/document/d/15c89-PyMc-QnSHglG1hf_Tub5lMkEcVYALL0rm5-LvA/edit#)
- Check it matches [the design](https://user-images.githubusercontent.com/1413534/92820570-03bacc00-f3c2-11ea-831d-d0096019517d.png)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/3075

## Screenshots
![image](https://user-images.githubusercontent.com/1413534/92954303-1e8d4f00-f45b-11ea-904d-fa822fda13b0.png)
